sos/report/plugins/ceph_mon.py

```diff
@@ -90,6 +89,31 @@ def setup(self):
             "ceph %s --format json-pretty" % s for s in ceph_cmds
         ], subdir="json_output", tags="insights_ceph_health_detail")
 
+        ceph_tell_cmds = [
+            "mon_status"
+        ]
```

Comment:
Until you plan to enhance the list by some other subcommands in some future, this is bit overkill. But even that is ACK from me.